onnx_tf/handlers/backend/q_linear_mat_mul.py

```diff
@@ -0,0 +1,57 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("QLinearMatMul")
+class QLinearMatMul(BackendHandler):
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    a = tensor_dict[node.inputs[0]]
+    a_scale = tensor_dict[node.inputs[1]]
+    a_zero_point = tensor_dict[node.inputs[2]]
+    b = tensor_dict[node.inputs[3]]
+    b_scale = tensor_dict[node.inputs[4]]
+    b_zero_point = tensor_dict[node.inputs[5]]
+    y_scale = tensor_dict[node.inputs[6]]
+    y_zero_point = tensor_dict[node.inputs[7]]
+    y_dtype = y_zero_point.dtype
+
+    # reshape 1-D a_scale, a_zero_point, y_scale and
+    # y_zero_point so it can broadcast in arithmetic
+    # operations later
+    a_scale_shape = a_scale.get_shape().as_list()
+    if len(a_scale_shape) > 0 and a_scale_shape[0] > 1:
```

Comment:
Consider using "if a_scale_shape and a_scale_shape[0] > 1:", a bit more efficient than the current code. Same for line 31.

Reply:
Done, @chinhuang007 please review again, thanks.